bin/bundle

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+# Exit cleanly from an early interrupt
+Signal.trap("INT") { exit 1 }
+
+$: << File.expand_path(File.join("__FILE__", "..", "lib"))
```

Comment:
Even if ```$:``` is well known for many people as for LOAD_PATH, I recommend using ```$LOAD_PATH``` as it's more clear for everyone.